### ISSUE TITLE
eslint-plugin-matrix-org has not recently been published to npm

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#18537]
https://github.com/matrix-org/eslint-plugin-matrix-org vs https://www.npmjs.com/package/eslint-config-matrix-org

**Comment by jryans** [comment]
Yes, I intentionally stopped publishing it to NPM when changing the package name because it seemed internal and not that useful to have on NPM. Currently all the consumers depend on it via a Git dependency.

Anyway, publishing could of course be resumed if we think there's a use for it.

**Comment by turt2live** [comment]
I started using it as a dependency in a sponsored project is all. Is there a way to take it off npm, updating the readme to say to use git?

**Comment by jryans** [comment]
I am not aware of a way to remove things from NPM, it's effectively a permanent record... Probably the closest possible option would be [deprecating](https://docs.npmjs.com/cli/v7/commands/npm-deprecate) it, which would show a warning when installed. Or I guess we could publish a fake version of the package with a new README...?

**Comment by turt2live** [comment]
Deprecating and breaking it intentionally would be fine, or keeping it updated instead

**Comment by jryans** [comment]
https://github.com/matrix-org/eslint-plugin-matrix-org/issues/16 tracks a related concern that the install instructions don't match reality.

**Comment by jryans** [comment]
@turt2live There is now a package under the new name: https://www.npmjs.com/package/eslint-plugin-matrix-org. Is that enough for your needs, or is there more that should happen here?

**Comment by turt2live** [comment]
err, not sure why it got a new package but that's fine. https://www.npmjs.com/package/eslint-config-matrix-org should be deprecated/deleted to avoid confusion.
